### PR TITLE
LPS-69540 SF

### PR DESCRIPTION
**Pull request by inacionery**

### CODE REVIEW

**Comment by inacionery** [comment]
ci:retest

File: portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/BaseWorkflowTaskManagerTestCase.java [hunk 1]
```diff
@@ -118,15 +118,28 @@ protected BlogsEntry addBlogsEntry() throws PortalException {
...
+		DDMForm ddmForm = DDMStructureTestUtil.getSampleDDMForm();
+
+		ddmStructure = DDMStructureTestUtil.addStructure(
+			groupId, JournalArticle.class.getName(), ddmForm);
+
+		ddmTemplate = DDMTemplateTestUtil.addTemplate(
+			groupId, ddmStructure.getStructureId(),
+			PortalUtil.getClassNameId(JournalArticle.class));
+
+		structurePK = ddmStructure.getPrimaryKey();
```
Comment (marcellustavares) [inline portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/BaseWorkflowTaskManagerTestCase.java]: @inacionery please rename to ddmStructurePK

File: portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java [hunk 2]
```diff
@@ -154,6 +154,28 @@ public void testAssignApproveWorkflowBlogsEntryAsPortalContentReviewer()
...
+
+		completeWorkflowTask(siteAdminUser, "reject");
+
+		blogsEntry = BlogsEntryLocalServiceUtil.getBlogsEntry(
+			blogsEntry.getEntryId());
+
+		Assert.assertEquals(
+			WorkflowConstants.STATUS_PENDING, blogsEntry.getStatus());
+
+		deactiveWorkflow(BlogsEntry.class.getName(), 0);
```
Comment (marcellustavares) [inline portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java]: can we rename this to `deactivateWorkflow` and `activateWorkflow`? As an action?

File: portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java [hunk 3]
```diff
@@ -100,7 +100,126 @@ public void testApproveJournalArticleAsAdmin() throws Exception {
...
+		throws Exception {
+
+		JournalFolder folder = addJournalFolder();
+
+		long folderId = folder.getFolderId();
+
+		updateJournalFolder(
+			folderId, 0, JournalFolderConstants.RESTRICTION_TYPE_WORKFLOW);
+
+		activeSingleApproverWorkflow(
```
Comment (marcellustavares) [inline portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java]: please change to `activateSingleApproverWorkflow`.

File: portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java (hunk 2 above)
Comment (vicnate5) [inline portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java]: It does sound better as "activate" and "deactivate"

Comment (marcellustavares) [inline portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java]: could you send a follow up @vicnate5?

Comment (vicnate5) [inline portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java]: @marcellustavares Yes

Comment (marcellustavares) [inline portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java]: thx

Comment (inacionery) [inline portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/runtime/integration/impl/internal/test/WorkflowTaskManagerImplTest.java]: hey @vicnate5 i'll send this changes, could you remove the Poshi tests on Master

**Pull request merged**